blackbox/docs/sql/statements/create_table.txt

```diff
@@ -47,6 +47,10 @@ where ``column_constraint`` is::
                             FULLTEXT [ WITH ( analyzer = analyzer_name ) ]  }
     }
 
+where ``storage_option`` is::
+
+    STORAGE WITH ( storage_option = value_expression [, ... ] )
```

Comment:
shouldn't we point to what the available storage options are? (currently only one of course)